src/app/components/work-item-quick-add/work-item-quick-add.component.less

```diff
@@ -0,0 +1,128 @@
+@import (reference) '../../../assets/stylesheets/_base';
```

Comment:
How come this file was added back to the repo?
@SMahil you need to more careful about your rebase and resolving conflicts. You always bring back old files. Please verify your conflict resolves with someone else everytime you do it.
@vikram-raj and @nimishamukherjee I see your approval there, but I believe the rebase happened after the approval so can't really blame. 

This is not the first time, so we need to be careful about rebasing and resolving conflicts. @SMahil I recommend you to take a proper git class from somewhere.